### PR TITLE
Security:register,login,logout,profile,profileedit.

### PR DESCRIPTION
**Pull request by Madaless**
Validation:user,predefied. Typo Errors

### CODE REVIEW

File: src/main/java/com/example/event_manager/configuration/WebSecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            .disable();
+  }
+
+  @Override
+  protected void configure(final AuthenticationManagerBuilder auth) throws Exception {
+    auth.userDetailsService((UserDetailsService) userService).passwordEncoder(passwordEncoder());
+  }
+
+  @PostConstruct
+  private void addAdmin() {
```
Comment (adamzareba) [inline src/main/java/com/example/event_manager/configuration/WebSecurityConfig.java]: remove this method and load users via migration

File: src/main/java/com/example/event_manager/exception/EventNotFoundException.java [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-
-  public EventNotFoundException(String msg) {
-    super(msg);
-  }
-
-  public EventNotFoundException() {
-  }
-
-}
+//package com.example.event_manager.exception;
```
Comment (adamzareba) [inline src/main/java/com/example/event_manager/exception/EventNotFoundException.java]: delete it if not needed

File: src/main/java/com/example/event_manager/configuration/WebSecurityConfig.java (hunk 1 above)
Comment (Madaless) [reply src/main/java/com/example/event_manager/configuration/WebSecurityConfig.java]: ho to do it since all the passwords are encoded ?

Comment (adamzareba) [reply src/main/java/com/example/event_manager/configuration/WebSecurityConfig.java]: just insert hashed passwords in migration script

**Pull request merged**